src/contexts/LayoutProvider.jsx

```diff
@@ -1,8 +1,8 @@
-import { useReducer } from "react";
+import { useReducer, createContext } from "react";
 
 import PropTypes from "prop-types";
 
-import LayoutContext from "./layoutContext";
+export const LayoutContext = createContext({ state: { open: false }, dispatch: () => {} });
```

Comment:
no need to export context as such like we can create a hook for this context , please refer AuthProvider file